sky/adaptors/azure.py

```diff
@@ -14,19 +17,40 @@
 
 _session_creation_lock = threading.RLock()
 
+# There is no programmatic way to get current account details anymore.
+# https://github.com/Azure/azure-sdk-for-python/issues/21561
+
+
+def _get_account():
+    result = run('az account show -o json',
+                 stdout=subprocess.PIPE,
+                 stderr=subprocess.PIPE)
+
+    if result.returncode != 0:
+        error_message = result.stderr.decode()
+        print(f'Error executing command: {error_message}')
```

Comment:
nit: could we include those error message into the error raised instead of print it?